src/cace/amm/semtype.c

```diff
@@ -94,10 +94,30 @@ int amm_type_set_use_ref_move(amm_type_t *type, ari_t *name)
     amm_semtype_use_init(semtype);
     ari_set_move(&(semtype->name), name);
 
+    int failcnt = 0;
```

Comment:
I think that this setter should _not_ bind the type object based on name. The function docs for these setters should be clear about this if they are not already.

There is separate logic in the agent to do type binding. If there is some issue with that it should be fixed in `refda/binding.c`.

Reply:
Looking at this again, maybe it's as simple as calling `refda_binding_semtype_use` from the generated CAMP code.

Reply:
OK, updated the FIXME in `binding.c` to handle operands for an `OPER`.